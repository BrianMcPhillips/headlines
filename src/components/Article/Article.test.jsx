import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './Article';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('renders Article', () => {
    const { asFragment } = render(<Article 
      urlToImage= "http://placecage.com/200/200"
      title= "Best Article"
      description= "The best article"
      author= "Best Article Writer"
      url= "https://www.bestarticle.com"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
