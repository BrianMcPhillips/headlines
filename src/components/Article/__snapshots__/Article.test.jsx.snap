// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="Best Article"
      src="http://placecage.com/200/200"
    />
    <h2>
      Best Article
    </h2>
    <h4>
      The best article
    </h4>
    <p>
      Best Article Writer
    </p>
    <a
      href="https://www.bestarticle.com"
    >
      View Article
    </a>
  </div>
</DocumentFragment>
`;
